evalutils/io.py

```diff
@@ -137,7 +137,7 @@ class ImageIOLoader(ImageLoader):
     @staticmethod
     def load_image(fname):
         with open(fname, "rb") as f:
-            with get_reader(f, mode="i", as_gray=True) as r:
```

Comment:
Why was `as_gray` removed here?

Reply:
![image](https://github.com/comic/evalutils/assets/29143120/c401bc1b-7e87-4481-b01f-55b2e9dfc2e5)

Reply:
This is a call to function get_reader from image_io see: https://imageio.readthedocs.io/en/stable/reference/userapi.html#imageio.v2.get_reader
where mode is:
mode : {'i', 'I', 'v', 'V', '?'}
        Used to give the reader a hint on what the user expects (default "?"):
        "i" for an image, "I" for multiple images, "v" for a volume,
        "V" for multiple volumes, "?" for don't care.

We don't/can't set the Pillow mode, And ImageIOLoader is only used in tests.

Reply:
`ImageIOLoader` is part of the public API (https://comic.github.io/evalutils/modules.html#evalutils.io.ImageIOLoader), challenge admins are able to use it in their own evaluation methods. We have to keep the return type the same!